react/src/components/molecules/FooterLinks/index.js

```diff
@@ -4,19 +4,19 @@ import './style.css';
 import { SidebarHeading } from '../../../index';
 
 const FooterLinks = (footerLinks) => (
-  <section className="ma__footer-links">
+  <nav className="ma__footer-links" aria-label="footer links">
     {
       footerLinks.items.map((footerLinksNav, i) => (
         /* eslint-disable-next-line react/no-array-index-key */
         <FooterLinksNav {...footerLinksNav} showNavHeading={footerLinks.showNavHeading} key={`footerLinksNav_${i}`} index={i} />
       ))
     }
-  </section>
+  </nav>
 );
 
 
 const FooterLinksNav = (footerLinksNav) => (
-  <nav aria-labelledby={footerLinksNav.id} className={`ma__footer-links__items${footerLinksNav.showNavHeading ? ' ma__footer-links__items--heading' : ''}`}>
+  <div className={`ma__footer-links__items${footerLinksNav.showNavHeading ? ' ma__footer-links__items--heading' : ''}`}>
```

Comment:
Hi @ygannett - could you clarify here? There's a classname on this wrapper, so if I remove the `<div>` I don't know where I'd put that classname. I assume I'd wrap the `<SidebarHeading>`/`<h3>` and the `<ul>` with a `<React.Fragment>`, but I don't believe I can put a classname on that.

Reply:
You see I put the `class` on `nav` on `ul` in above comment. Since `h3` is not visible, this doesn't cause any display issue.  `ul.ma__footer-links__items` would maintain the current positioning. You can say that the `ul` turns the wrapper used to be `nav`.

I'm looking at output markup and not really looking from React side.  If this would cause any troubles in React, please let me know. I'm not so familiar with React, so I might not be aware of certain things from its aspect. I apologize for that, but please don't hesitate to point them out.